plugins/chat/app/services/chat/channel_view_builder.rb

```diff
@@ -226,6 +227,10 @@ def fetch_thread_participants(threads:, **)
         ::Chat::ThreadParticipantQuery.call(thread_ids: threads.map(&:id))
     end
 
+    def update_channel_last_viewed_at(channel:, guardian:, **)
+      channel.membership_for(guardian.user)&.update!(last_viewed_at: Time.zone.now)
```

Comment:
```suggestion
      channel.membership_for(guardian.user)&.update!(last_viewed_at: Time.current)
```

Reply:
I really don't get why this is coming up now when we've used `Time.zone.now` so much more than `Time.current`...

Reply:
If you really want me to I will change it, I mostly don't see a reason to since they are functionally equivalent and we've used `Time.zone.now` way more.

Reply:
I dont care